docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     depends_on:
       - db
       - redis
+      - elasticsearch
```

Comment:
:thinking: Should these be here?

Reply:
Nope. I was sort of working on two branches at the same time because Docker on macOS is slooooooow and this snuck in.